lib/vanagon/platform/windows.rb

```diff
@@ -34,6 +34,143 @@ def package_name(project)
         "#{project.name}-#{project.version}-#{project.release}-#{@architecture}.msi"
       end
 
+      def add_repository(definition)
+        definition = URI.parse(definition)
+
+        ps1_command = install_chocolately_ps1 (definition)
+
+
+        commands = ["cat <<EOF; #{ps1_command}; EOF > install-chocolatey.ps1"]
+
+        #install_chocolately_on_host(definition)
+        puts "Now doing Add_Repo #{definition}"
+
+        if definition.scheme =~ /^(http|ftp)/
+          # use choco to install the package.
+          # choco source add -n=bob -s "https://somewhere/out/there/api/v2/"
+          puts "Adding Repo #{definition}"
+          commands << "echo choco add -s #{definition}"
```

Comment:
The command being appended doesn't quite line up with the comment above it; is that intentional?